### ISSUE TITLE
Lock screen time out not being respected

### ISSUE THREAD

**Issue by pelo88** [durka/HallMonitor#57]
I have my lock screen set to come on 10 minutes after the screen shuts off but when I close my lid with hall monitor and the sview disappears it locks right away forcing me to enter my pin when I open the lid. Also for whatever reason despite my trust location here at work the lock screen is still coming on while I'm here and close and open the lid. I use CM13 on my note 3. Love this tho. Just wish it used my whole sview. The notes have larger sview flip covers than the so.

**Comment by pelo88** [comment]
S4

**Comment by pelo88** [comment]
Yes I can confirm smart lock no longer works, it never unlocks my device automatically since I installed this.

**Comment by pelo88** [comment]
It's worse than I thought. Even without closing the flip cover just having hall monitor installed none of my smart lock trusted things, Bluetooth or GPS locations, unlock my screen.

**Comment by TinoDidriksen** [comment]
I can confirm this. Smart Lock and other lock screen settings are completely ignored when HallMonitor is installed (even using @habeIchVergessen latest Framework Test release).
